robot/basestation/static/js/elements/navigation/navigation.js

```diff
@@ -54,137 +105,688 @@ $(document).ready(() => {
-    let hasAntennaParams = true
-    antenna_latitude.get(function (val) {
-      if (val != null) {
-        $('#antenna-latitude').text(val)
-        antenna_longitude.get(function (val) {
-          if (val != null) {
-            $('#antenna-longitude').text(val)
-            antenna_start_dir.get(function (val) {
-              if (val != null) {
-                $('#antenna-start-dir').text(val)
-                appendToConsole(
-                  'Antenna goal parameters already set, displaying antenna directions'
-                )
-                $('#antenna-inputs').hide()
-                $('#antenna-unchanging').show()
-              } else {
-                appendToConsole(
-                  'One or more antenna parameters is missing, if you would like antenna directions then please input them'
-                )
-                $('#antenna-inputs').show()
-                $('#antenna-unchanging').hide()
-              }
-            })
-          } else {
-            appendToConsole(
-              'One or more antenna parameters is missing, if you would like antenna directions then please input them'
-            )
-            $('#antenna-inputs').show()
-            $('#antenna-unchanging').hide()
+var navQueue = new NavigationQueue()
+var count = 0
+
+  function initNavigationPanel () {    
+
+    antenna_latitude.get(function(lat){
+      if(lat != null){
+        antenna_longitude.get(function(long){
+          if(long != null){
+            antenna_start_dir.get(function(dir){
+              if(dir != null){
+                appendToConsole('Antenna parameters already set')
+                    got_antenna_pos.set(true)
+                
+                $.get({
+                  url : '/navigation/cached_content/antenna_stats',
+                  success: function(result){
+                      
+                        
+                        $('#antenna-stats-lat').text(lat.toFixed(6))
+                        $('#antenna-stats-long').text(long.toFixed(6))
+                        $('#antenna-stats-heading').text(dir)
+                         if(! jQuery.isEmptyObject(result)) {
+
+                        $('#antenna-stats-dist-to-rover').text(JSON.parse(result).distance) 
+                        $('#antenna-stats-rec-angle').text(JSON.parse(result).heading)
+                          }
+                          else{
+                            $('#antenna-stats-dist-to-rover').text("----") 
+                            $('#antenna-stats-rec-angle').text("----")
+                            
+                          }
+                  },
+
+                })
+                $.get({
+                  url : '/navigation/cached_content/antenna_modal',
+                  success: function(result){
+                        
+                       
+                       let data =  $('#antenna-select-lat-format').detach()
+                       let data2 =  $('#antenna-select-long-format').detach()
```

Comment:
data2

Reply:
Please read the https://github.com/space-concordia-robotics/robotics-prototype/blob/master/CONTRIBUTING.md

Reply:
Yeah my bad this was just pure laziness lol.